pkg/engine/internal/proto/expressionpb/unmarshal.go

```diff
@@ -165,6 +165,8 @@ func (e *LiteralExpression) UnmarshalPhysical(from physical.Expression) error {
 		e.Kind = &LiteralExpression_BytesLiteral{}
 	case types.StringListLiteral:
 		e.Kind = &LiteralExpression_StringListLiteral{}
+	default:
+		return fmt.Errorf("invalid literal type for %v (%v): %T", from, ty, ty)
```

Comment:
nit: why print this twice? `%v (%v)` sorry i am not able to figure out what the output would look like